### PR TITLE
improve footer

### PR DESCRIPTION
**Pull request by mdrianislam0or1**
### Description
 Improve footer styles in map view - Desktop view #171 

#### Screenshots
fix#171

### CODE REVIEW

**Review by Israel-Laguan — APPROVED** [review]

**Review by e-azocar — APPROVED** [review]

File: src/pages/Delivery/Delivery.layout.jsx [hunk 1]
```diff
@@ -1,23 +1,15 @@
...
+import { Fragment } from 'react'
 import assistant from '../../assets/assistant.png'
 import chat from '../../assets/chat.png'
 import gps from '../../assets/gps-icon.png'
-import { Fragment } from 'react'
+import { Text } from '../../components'
+import {
+  ChatBtn, ChatBtnContainer, FooterMap,
+  FooterMapAddress, HeaderMap, HeaderSubTitle, HeaderText, HeaderTitle, MapContainer, MapLayoutContainer
+} from './Delivery.styled'
```
Comment (heliomar-pena) [inline src/pages/Delivery/Delivery.layout.jsx]: I prefer one line for each one

```suggestion
import {
  ChatBtn, 
  ChatBtnContainer, 
  FooterMap,
  FooterMapAddress, 
  HeaderMap,
  HeaderSubTitle, 
  HeaderText, 
  HeaderTitle, 
  MapContainer, 
  MapLayoutContainer
} from './Delivery.styled'
```

Please check if is possible to run Prettier

**Review by heliomar-pena — CHANGES_REQUESTED** [review]
Hmm please check code style, the rest looks good

**Comment by angelik0828** [comment]
@mdrianislam0or1 

1. The text used in the localization is not bold and is not proportionally sized.
2. The text used in "estimated time of arrival" is not bold and should not be as large
3. The text "contact us" should be blank and in normal type.

image

File: src/pages/Delivery/Delivery.layout.jsx [hunk 2]
```diff
@@ -66,12 +66,12 @@ const Delivery = ({
           >
             <img src={gps} width="22px" height="31px" alt="Gps icon" style={{ margin: '10px'}} />
           </FooterMapAddress>
-          <FooterMapAddress cols={'0 0 50%'} txt={'Tiempo estimado de llegada'}>
-            <Text medium bold margin="8" style={{ textTransform: 'lowercase', color:'#003B62',fontWeight:'700'}}>
+          <FooterMapAddress style={{fontSize:'25px'}} cols={'0 0 50%'} txt={'Tiempo estimado de llegada'}>
+            <Text medium bold margin="8" style={{ textTransform: 'lowercase',fontSize:'23px', color:'#003B62',fontWeight:'700'}}>
```
Comment (heliomar-pena) [inline src/pages/Delivery/Delivery.layout.jsx]: please left enough space to do more readable your code
```suggestion
            <Text medium bold margin="8" style={{ textTransform: 'lowercase', fontSize: '23px',  color: '#003B62', fontWeight: '700' }}>
```

**Review by Israel-Laguan — APPROVED** [review]

**Comment by angelik0828** [comment]
@mdrianislam0or1 I want to ask if the change was made?, I didn't see any capture

**Review by angelik0828 — APPROVED** [review]

**Pull request merged**